app/extensions/brave/locales/en-US/app.properties

```diff
@@ -237,6 +237,8 @@ streetAddress=Street Address
 submit=Submit
 tabsSuggestionTitle=Tabs
 topSiteSuggestionTitle=Top Site
+torrentBlockedInTor=Torrents are not allowed in tor private tabs.
```

Comment:
Can you change this to `For your privacy, torrents are blocked in private tabs when Tor is enabled.`